net/wireguard/files/wireguard.sh

```diff
@@ -161,6 +140,13 @@ proto_wireguard_setup() {
     exit 1
   fi
 
+  # endpoint dependency
+  for wg_endpoint in $(wg show ${config} endpoints | grep -v '(none)'); do
+    ip=$(echo ${wg_endpoint} | sed 's/.*\t\[\?\([0-9a-f\.:]\+\)\]\?:[0-9]\{1,5\}/\1/')
```

Comment:
There's a cleaner way to do this without the regex:

```bash
wg show "${config}" endpoints | while IFS=$'\t:' read -r key ip port; do
    [ -n "${port}" ] || continue
    proto_add_host_dependency "${config}" "${ip}"
done
```
(No need to declare key, ip, and port as `local` since that loop actually is in a subshell.)